src/Selenium2Driver.php

```diff
@@ -214,6 +214,23 @@ public static function getDefaultCapabilities()
         );
     }
 
+    /**
+     * Checks if the WebDriver session is W3C compatible.
+     *
+     * @return bool
+     *
+     * @throws UnknownCommand
```

Comment:
This is wrong. This method will never throw this exception (it precisely catches it)